src/app/Profile/dress/page.tsx

```diff
@@ -0,0 +1,299 @@
+          className={`w-20 h-7 flex flex-shrink-0 items-center justify-center rounded-lg font-medium
+            ${activeTab === tab.id
+              ? "bg-gradient-to-b from-[#ECEFF6] to-[#3B83A9] text-[#262D37]" // Style for active tab
+              : "bg-gradient-to-b from-[#092B44] via-[#625B87] to-[#D2CAFF]  text-white " // Style for inactive tab
+            }`}
+        >
+          {tab.label}
+        </button>
+      ))}
+      
+    </div>
+  );
+};
+
+type element = {
+  id: string;
+  imageSrc: string;
+  isLocked: boolean;
+  colors: number;
+}
+
+type item = element[]
+
+
+function ItemsGrid({ items, activeTab, activeItemId, setActiveItemId, handlePartSelection }: { items: Record<string, item>, activeTab: string,
+  activeItemId: string|null, setActiveItemId: (tab:string|null) => void, handlePartSelection: (part:string, id:string) => void }) {
+
+  const displayedItems = (() => {
+    if (!activeItemId) {
+      return items[activeTab];
+    }
+
+    const activeItem = items[activeTab]?.find((item) => item.id === activeItemId);
+
+    if (activeItem && activeItem.colors > 1) {
+      return Array.from({ length: activeItem.colors }, (_, index) => ({
+        ...activeItem,
+        id: `${activeItem.id}-${index + 1}`,
+        imageSrc: `/model/${activeTab}/${activeItem.id}/${activeItem.id}-${index + 1}.webp`,
+      }));
+    }
+
+    return items[activeTab];
+  })();
+
+  const handleClickItem = (id: string, imgSrc: string) => {
+    setActiveItemId(id);
+    handlePartSelection(activeTab, imgSrc);
+  }
+
+  return (
+    <div className="grid grid-cols-3 w-full justify-center justify-items-center bg-[#ECF0F6]/80 h-60 rounded-b-lg"
+        style={{
+        overflow: "auto", 
+        scrollbarWidth: "none", 
+        msOverflowStyle: "none", 
+        WebkitOverflowScrolling: "touch", 
+        overflowY: "scroll"
+      }}>
+      {displayedItems && displayedItems.map((item: element) => ( 
+        <div 
+          key={item.id}
+          className={`flex flex-col items-center justify-center bg-[#36465F]/30 w-24 h-14 my-2.5 mx-2 cursor-pointer rounded-lg`}
+          style={{ minHeight: "100px" }}
+          onClick={() => handleClickItem(item.id, item.imageSrc)}
+          >
+          {/* Render image */}
+          <img src={item.imageSrc} alt={`Item ${item.id}`} className="w-full h-full object-contain" />
+        </div> 
+      ))}
+    </div>
+  );
+};
+
+export default function Dress() {
+  const [activeTab, setActiveTab] = useState("skin")
+
+  const [activeItemId, setActiveItemId] = useState<string|null>(null)
+
+  const [selectedParts, setSelectedParts] = useState<{
+    skin: string | null;
+    hair: string | null;
+    eyebrow: string | null;
+    eye: string | null;
+    nose: string | null;
+    mouth: string | null;
+    shirt: string | null;
+    clothes: string | null;
+    pant: string | null;
+    shoes: string | null;
+  }>(() => {
+    const savedAvatar = localStorage.getItem("selectedParts");
```

Comment:
add type for `savedAvatar` because "Unsafe return of a value of type `any`." in line 154